jukebox_radio/users/views/user_profile_view.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.mixins import LoginRequiredMixin
+
+from jukebox_radio.core.base_view import BaseView
+
+User = get_user_model()
+
+class UserGetProfileView(BaseView, LoginRequiredMixin):
+  def get(self, request, **kwargs):
+    """
+    Get user profile.
+    """
+
+    return self.http_response_200({
+      "user": {
+        "username": request.user.username,
+        "profile_image": request.user.profile_image,
```

Comment:
Here profile image will be read like: `request.user.userprofile.profile_image`. Same thing with `description` and `website`.